recipes/skprogs-python/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "skprogs" %}
+{% set version = "0.2" %}
+
+package:
+  name: {{ name|lower }}-python
+  version: {{ version }}
+
+source:
+  url: https://github.com/dftbplus/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: a16cc91cfa702b2e40b61bbe218fd43d9225469c6e90b56ab04b89838916f9bf
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - collectspinw = sktools.scripts.collectspinw:main
+    - collectwavecoeffs = sktools.scripts.collectwavecoeffs:main
+    - skdiff = sktools.scripts.skdiff:main
+    - skgen = sktools.scripts.skgen:main
+    - skmanip = sktools.scripts.skmanip:main
+  script: "{{ PYTHON }} -m pip install ./sktools -vv"
+
+requirements:
+  host:
+    - python >=3.9
+    - pip
+  run:
+    - python >=3.9
+    - numpy
+    - scipy
+
+test:
+  imports:
+    - sktools
+  commands:
+    - collectspinw -h
+    - collectwavecoeffs -h
+    - skdiff -h
+    - skgen -h
+    - skmanip -h
+
+about:
+  home: https://github.com/dftbplus/{{ name }}
```

The jinja variable just obscures the URL. No need to use it here.

```suggestion
  home: https://github.com/dftbplus/skprogs
```

---

Sure, no problem.